hack/manifest-gen/go.mod

```diff
@@ -7,5 +7,5 @@ require (
 	github.com/spf13/cobra v1.0.0
 	helm.sh/helm/v3 v3.3.1
 	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/kustomize/kyaml v0.7.1
+	sigs.k8s.io/kustomize/kyaml/v3/v3 v3.3.1
```

Comment:
Last `kyaml` version seems to be `v0.8.1`, I don't understand why renovate wants to upgrade to `v3.3.1` (which seems to be a valid tag though)